detox/src/devices/drivers/EmulatorDriver.js

```diff
@@ -19,7 +19,8 @@ class EmulatorDriver extends AndroidDriver {
   }
 
   async _fixEmulatorConfigIniSkinName(name) {
-    const configFile = `${os.homedir()}/.android/avd/${name}.avd/config.ini`;
+    const sdkHome = process.env.ANDROID_SDK_HOME || os.homedir();
+    const configFile = path.normalize(`${sdkHome}/.android/avd/${name}.avd/config.ini`);
```

Comment:
I am not entirely sure, but isn't `sdkHome` supposed to include that `.android` as well?
I made a quick look up in: https://stackoverflow.com/questions/23042638/how-do-i-set-android-sdk-home-environment-variable
And among examples of how can `ANDROID_SDK_HOME` look like, I see strings like that:

```
C:\Users\<USERNAME>\.android\
```

Hence, it is meant to include `.android` dir path within. So why do you append `.android` again to it?

Reply:
@pettersamuelsen, ping.

Reply:
I have checked that stuff on SO once again, and it seems that the OP made a typo in the answer there.

Nevertheless, I dug into the docs:  https://developer.android.com/studio/command-line/variables

As far, as I understand, what really matters, is this environment variable: `ANDROID_EMULATOR_HOME`. Experimentally verified:

```bash
> emulator -list-avds
# Nexus_4_4
# Nexus_4_API_25
# Nexus_5X_API_23
# Nexus_5X_API_24
# Nexus_5X_API_25
# Nexus_5X_API_26
# Nexus_5X_API_28
# Nexus_5_0
```

and with it:

``` bash
> ANDROID_EMULATOR_HOME=/tmp emulator -list-avds
#
```

The same goes for `ANDROID_AVD_HOME`. It affects `emulator` executable.

But as for `ANDROID_SDK_HOME` - alas, I have not found any effect on the emulator boot process. Looks like it is ignored in this regard.

So, maybe you should use `ANDROID_AVD_HOME` + `ANDROID_EMULATOR_HOME` in your logic there?

Reply:
@pettersamuelsen thoughts on next steps?